scripts/create-tree.ts

```diff
@@ -20,13 +20,7 @@ async function main() {
 
   console.log(`Creating a merkle tree and proofs for ${options.input}...`);
```

Comment:
Hmm, I think this would be a bit overkill tbh. If we would have different log transports and log levels then I agree but these scripts are pretty lightweight and won't be used that often.